mongodb-storage/src/main/java/io/lumeer/mongodb/MongoDbStorage.java

```diff
@@ -215,8 +216,8 @@ public boolean hasCollection(final String collectionName) {
    }
 
    @Override
-   public boolean collectionHasDocument(final String collectionName, final String filter) {
-      return database.getCollection(collectionName).find(BsonDocument.parse(filter)).limit(1).iterator().hasNext();
+   public boolean collectionHasDocument(final String collectionName, final DataFilter filter) {
+      return database.getCollection(collectionName).find((Bson) filter.get()).limit(1).iterator().hasNext();
```

Comment:
The following code would probably be better in order to avoid casting:
```java
...find(filter.<Bson>get())...
```
But it is not very important and it can stay as it is.